### ISSUE TITLE
Can't open sidebar on firefox 火狐无法使用侧边栏

### ISSUE THREAD

**Issue by ZreXoc** [ChatGPTBox-dev/chatGPTBox#801]
**Describe the bug**
**问题描述**
Can't open sidebar on firefox

**To Reproduce**
**如何复现**
1. 打开网页（如bing）
2. 右键菜单选择打开侧边栏
3. 无事发生，控制台无报错

**Expected behavior**
**期望行为**
正常打开侧边栏

**Please complete the following information):**
**请补全以下内容**
 - OS: Linux
 - Browser: Mozilla Firefox
 - Extension Version:  131.0

**Additional context**
**其他**
有什么其他信息需要我提供吗？

**Comment by josStorer** [comment]
火狐还没有支持相关的API

**Comment by antonovkz** [comment]
This is not true. Extensions like TabReborn, Page Assist, and many others handle this task perfectly.
